### DISCUSSION TITLE
FinOps PowerBI report

### DISCUSSION THREAD

**Discussion by NavidKhan8** [CloudMonitorFinOps/reporting#64] [General]
I was trying to follow the repo https://github.com/CloudMonitorFinOps/reporting/tree/main/FinOps for PowerBI reports but this repo doesn't have the readme and when tried to open pbix files most of the files are unable to open with an error "owner has not shared the data model" @rodneyjoyce @DanMarshall909  @komzy96  @zhang-andrew  - Would you be able to help? [CloudMonitor FinOps Dataset.pbix] was able open it but the visual doesn't have much information. Do we have any screenshots to refer these reports?

**Comment by rodneyjoyce** [comment]
Apologies, I did not get notified on this question! We've added a ReadMe and lots of other info - are you still having the same issue?
